libs/utils/trace.py

```diff
@@ -729,6 +729,43 @@ def _loadFunctionsStats(self, path='trace.stats'):
 
         return len(self._functions_stats_df) > 0
 
+    def _cropToXTimeRange(self, data):
+        """
+        Crop a dataframe containing FTrace events to the X time range specified
+        by the user through setXTimeRange().
+
+        When cropping a series it might be necessary to include a first and
+        last element with timestamps equal to the boundaries of the X time
+        range.
+
+        :param data: series to be cropped
+        :type data: :mod:`pandas.Series`
+        """
+        if not isinstance(data, pd.Series):
+            raise ValueError('cropping supported only for pandas.Series '
+                             'objects!')
```

Comment:
let's keep on same line... or better defined a msg=''cropping..." in the previous line ;-)